sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs

```diff
@@ -2091,7 +2091,13 @@ public partial class SiteFunctionResource : Azure.ResourceManager.ArmResource
         public virtual System.Threading.Tasks.Task<Azure.Response> DeleteFunctionSecretAsync(string keyName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.ResourceManager.AppService.SiteFunctionResource> Get(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.AppService.SiteFunctionResource>> GetAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("This property is obsolete and will be removed in a future release, please use `GetFunctionKeysAsDictionary` instead", false)]
         public virtual Azure.Response<Azure.ResourceManager.AppService.Models.AppServiceConfigurationDictionary> GetFunctionKeys(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<System.Collections.Generic.IReadOnlyDictionary<string, string>> GetFunctionKeysAsDictionary(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
This is not a good mitigation solution which has too many customization code.
You can use `directive` to fix the response of these 2 operations, rename the function names of these to operations:
1. GetFunctionKeys -> GetSiteFunctionKeys
2. GetFunctionKeysSlot -> GetSiteSlotFunctionKeys
Update the customization code that only contains 2 old methods which can just return empty (can new a AppServiceConfigurationDictionary instance from the Dictionary result? If yes try it, call new method and convert the Dictionary result to AppServiceConfigurationDictionary).
Then regen the sdk.

Reply:
followed dapeng's way